src/db/tables/profiles.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import initKnex from 'knex'
+import knexConfig from '../knexfile.js'
+
+const knex = initKnex(knexConfig)
+
+/**
+ * @param {number} onerepProfileId
+ */
+async function getProfileDetails (onerepProfileId) {
+  await knex('profiles')
+    .select('onerep_profile_id',
+      'first_name',
+      'last_name',
+      'city_name',
+      'state_code',
+      'date_of_birth',
+      'created_at'
+    )
+    .where('onerep_profile_id', onerepProfileId)
+}
+
+/**
+ * @param {number} onerepProfileId
+ * @param {import("../../app/functions/server/onerep.js")} profileData
+ */
+async function setProfileDetails (onerepProfileId, profileData) {
+  await knex('profiles')
+    .insert({
+      onerep_profile_id: onerepProfileId,
+      first_name: profileData.first_name,
+      last_name: profileData.last_name,
+      city_name: profileData.addresses[0].city_name,
+      state_code: profileData.addresses[0].state_name,
```

Comment:
state?

Reply:
In my `profile` table, these values aren't set
<img width="751" alt="Screenshot 2023-07-11 at 4 20 50 PM" src="https://github.com/mozilla/blurts-server/assets/42494162/6fd2e954-1001-4fac-af75-7d573e909ada">